snippets/bootstrap-migration/main/header/top.liquid

```diff
@@ -38,11 +38,8 @@
                         <li class="list-inline-item">
                             {% raw %}{{ customer.firstName }} {{ customer.lastName }}{% endraw %}
                         </li>
-                        <li class="list-inline-item dropdown" uib-dropdown auto-close="mouseleave" aria-haspopup="true">
-                            <a id="account-dropdown" href="{{ 'account' | absolute_url }}" uib-dropdown-toggle vc-dropdown-trigger="mouseenter">{{ 'layout.customer.account' | t }}&ensp;<i class="fa fa-angle-down text-primary"></i></a>
-                            <div class="dropdown-menu dropdown-menu-right mt-0" role="menu" uib-dropdown-menu aria-labelledby="account-dropdown">
-                                    {{ 'layout.customer.log_out' | t | customer_logout_link | replace: 'href', 'role="menuitem" class="dropdown-item font-weight-normal stretched-link" tabindex="-1" href' }}
-                            </div>
+                        <li class="list-inline-item">
+                            <vc-account-switch-migration class="d-inline-block"></vc-account-switch-migration>
```

Comment:
Is it a correct component name? Why it contains "migration"?

Reply:
Because we have to version of the same component: one for Bootstrap 3 and one for Bootstrap 4 ([see this](https://github.com/VirtoCommerce/vc-demo-theme-b2b/pull/192/files#diff-64514536a472db26e37b64f1f28f7ff8627c999bb874dbfe0cf0e3d339e62384R18-R27)). So we use this approach to use same controller. It already was used [here](https://github.com/VirtoCommerce/vc-demo-theme-b2b/blob/f95c1478151b8c8e979ad6fbb990af6ccd6523e5/assets/js/lists/add-to-list-button.js#L52-L67) and [here](https://github.com/VirtoCommerce/vc-demo-theme-b2b/blob/38ce70de6178aa71f5b90ef36cab78e4f0d34695/snippets/bootstrap-migration/product/type/common/product-action-buttons.liquid#L4) for example.